assignment/README.md

```diff
@@ -12,17 +12,36 @@ Applications are ultimately just tools. What problem or question does
 your application attempt to resolve or grapple with? How does your
 application speak to this problem/question?
 
+Explore the level of bus feeds to/from SEPTA rail stations (BSL, MFL and Regional Rail).
+Basically, the map would show the level (trips per day) of bus service around each rail station within a certain distance (plotted using MapBox api).
+A basic description:
+When the user specifies a search radius (ex: 10 mins of walking time or 5 minutes of biking time) and clicks on a rail station, the map would display all bus stops within that distance (as dots), and display the routes of buses that serve these bus stops (as polylines).
+The map would also show the number of trips available per day for these routes, which can be filtered by Weekdays, Saturdays and/or Sundays.
+The stations and bus lines could be color-coded based on the service frequency of bus lines.
+
+(Maybe, stretch goal) if I can figure out how the SEPTA api works, display the next bus location on these routes.
```

Comment:
This is a good stretch goal. Worth playing around with even if you ultimately decide against including it